src/api/models/nls/ncn_rebuild.go

```diff
@@ -33,7 +33,8 @@ type CreateRebuildWorkflowRequest struct {
 	Hosts          []string `json:"hosts"`
 	DryRun         bool     `json:"dryRun"`
 	SwitchPassword string   `json:"switchPassword,omitempty"`
-	WipeOsd        bool     `json:"wipeOsd,omitempty"`
```

Comment:
Are the `ZapOsds` and `WorkflowType` request parameters somewhat redundant? From our conversations, it sounded like the only difference between a storage rebuild vs. upgrade was that a rebuild would zap/wipe the OSDs. If there is no difference, it seems like we could get rid of one of them.

Reply:
My understanding is that zapping the osds is different than wiping them. So `ZapOsds` should be its own parameter. `WorkflowType` is needed to tell cray-nls which workflow it should run. There is a different procedure for upgrade vs. rebuild.

Reply:
To clarify, WipeOsd is not changing to ZapOsds. WipeOsds is being removed all together. ZapOsds is being added as a different parameter for a different purpose.

Reply:
That helps, thanks! It is a bit odd to have a `WorkflowType` field that can be either "rebuild" or "upgrade" in this struct called `CreateRebuildWorkflowRequest` that defines the parameters to be passed when creating a rebuild workflow, but I don't have a simple suggestion on addressing that.